src/pcapi/core/logging.py

```diff
@@ -90,6 +92,17 @@ def makeRecord(self, name, level, fn, lno, msg, args, exc_info, func=None, extra
     logging.Logger.makeRecord = makeRecord
 
 
+class JsonLogEncoder(json.JSONEncoder):
+    def default(self, obj):
+        if isinstance(obj, decimal.Decimal):
+            return float(obj)
+        if isinstance(obj, enum.Enum):
+            return str(obj)
+        if hasattr(obj, "id"):
+            return obj.id
```

Comment:
Mmh, peut-être mettre 
```suggestion
        if hasattr(obj, "id"):
            return f"{object: type(obj).__name__ with id: {obj.id}"
```
?